lib/rules/attributes-order.js

```diff
@@ -200,17 +206,27 @@ function create(context) {
     [ATTRS.UNIQUE, ATTRS.SLOT],
     ATTRS.TWO_WAY_BINDING,
     ATTRS.OTHER_DIRECTIVES,
-    ATTRS.OTHER_ATTR,
+    [ATTRS.OTHER_ATTR_DYNAMIC, ATTRS.OTHER_ATTR_STATIC],
     ATTRS.EVENTS,
     ATTRS.CONTENT
   ]
   if (context.options[0] && context.options[0].order) {
-    attributeOrder = context.options[0].order
+    attributeOrder = [...context.options[0].order]
   }
   const alphabetical = Boolean(
     context.options[0] && context.options[0].alphabetical
   )
 
+  for (const [index, item] of attributeOrder.entries()) {
```

Comment:
Please add an explanatory comment here:

```suggestion

  // expand `OTHER_ATTR` alias 
  for (const [index, item] of attributeOrder.entries()) {
```

Reply:
Added